awsiot/greengrasscoreipc/client.py

```diff
@@ -1088,14 +1088,14 @@ def activate(self, request: model.ValidateAuthorizationTokenRequest) -> concurre
         """
         return self._activate(request)
 
-    def get_response(self) -> concurrent.futures.Future:
+    def get_response(self):  # type: (...) -> concurrent.futures.Future[model.ValidateAuthorizationTokenResponse]
         """
         Returns a Future which completes with a result of ValidateAuthorizationTokenResponse,
         when the initial response is received, or an exception.
         """
         return self._get_response()
 
-    def close(self) -> concurrent.futures.Future:
+    def close(self):  # type: (...) -> concurrent.futures.Future[None]
         """
         Close the operation, whether or not it has completed.
 
```

Comment:
a few lines below this the `class GreengrassCoreIPCClient` is defined.

Could you please add a comment to the old client class, about how users might want to check out the newer easier to use client?

Reply:
Given that it is only a preview right now, I'm not sure we should, but certainly I could do that.